sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/MultiplexingFnDataReceiver.java

```diff
@@ -18,29 +18,32 @@
 
 package org.apache.beam.fn.harness.data;
 
+import com.google.common.collect.Iterables;
 import java.util.Collection;
-import org.apache.beam.fn.harness.fn.ThrowingConsumer;
 import org.apache.beam.sdk.fn.data.FnDataReceiver;
 
 /**
  * A {@link FnDataReceiver} which forwards all received inputs to a collection of {@link
- * ThrowingConsumer consumers}.
+ * FnDataReceiver receivers}.
  */
 public class MultiplexingFnDataReceiver<T> implements FnDataReceiver<T> {
-  public static <T> MultiplexingFnDataReceiver<T> forConsumers(
-      Collection<ThrowingConsumer<T>> consumers) {
+  public static <T> FnDataReceiver<T> forConsumers(
+      Collection<FnDataReceiver<T>> consumers) {
+    if (consumers.size() == 1) {
+      return Iterables.getOnlyElement(consumers)::accept;
```

Comment:
You should be able to return the single consumer and not a method reference to the `accept` method